thesis/behaviour.som

```diff
@@ -0,0 +1,45 @@
+class Behaviour usingPlatform: platform = Value (
```

Comment:
Ok, you might want to put them in a separate repo to avoid issues, since you seem to have sent the PR from master, I am worried that if i delete them, you might just miss them.